### PR TITLE
fix : Share of checkout new wording

### PR DESCRIPTION
**Pull request by Francois-Gomis**

### CODE REVIEW

File: etc/adminhtml/system.xml [hunk 1]
```diff
@@ -334,8 +334,9 @@
                         <comment><![CDATA[<b>Advanced.</b> Overrides the <code>customer_cancel_url</code> value of created payments]]></comment>
                         <config_path>payment/alma_monthly_payments/customer_cancel_url</config_path>
                     </field>
-                    <field id="share_checkout_enable" translate="label" type="select" sortOrder="17" showInDefault="1">
-                        <label>Enabled Share of Checkout</label>
+                    <field id="share_checkout_enable" translate="label comment" type="select" sortOrder="17" showInDefault="1">
```
Comment (syjust-alma) [inline etc/adminhtml/system.xml:337]: why did you add `comment` after `label` in `translate` attribute ?

**Review by syjust-alma — APPROVED** [review]

File: etc/adminhtml/system.xml (hunk 1 above)
Comment (Francois-Gomis) [reply etc/adminhtml/system.xml:337]: To also translate the comment.é
